MiniApp/Classes/Display/RealMiniAppView.swift

```diff
@@ -99,23 +100,29 @@ internal class RealMiniAppView: UIView {
         webView.configuration.userContentController.removeMessageHandler()
     }
 
-    func webView(_ webView: WKWebView, decidePolicyFor navigationAction: WKNavigationAction, decisionHandler: @escaping (WKNavigationActionPolicy) -> Void) {
-        if let requestUrl = navigationAction.request.url {
-            validateScheme(requestURL: requestUrl, decisionHandler: decisionHandler)
-        } else {
-            decisionHandler(.cancel)
-        }
-    }
-
-    func validateScheme(requestURL: URL, decisionHandler: @escaping (WKNavigationActionPolicy) -> Void) {
-        let schemeType = MiniAppSupportedSchemes(rawValue: requestURL.scheme!)
-        switch schemeType {
-        case .tel:
-            UIApplication.shared.open(requestURL, options: [:], completionHandler: nil)
-            decisionHandler(.cancel)
-        default:
-            decisionHandler(.allow)
+    func validateScheme(requestURL: URL, navigationAction: WKNavigationAction, decisionHandler: @escaping (WKNavigationActionPolicy) -> Void) {
+        if let scheme = requestURL.scheme {
+            let schemeType = MiniAppSupportedSchemes(rawValue: scheme)
+            switch schemeType {
+            case .about:
```

Comment:
What is the `about` scheme used for?

Reply:
the about scheme is the scheme used for js alert dialogs, for example. This one brought me some challenge to find out

Reply:
Ah, okay. Maybe you should add a comment so we don't delete it by accident?